src/nspawn/nspawn-settings.h

```diff
@@ -56,9 +56,19 @@ typedef enum SettingsMask {
         SETTING_CPU_AFFINITY      = UINT64_C(1) << 20,
         SETTING_RLIMIT_FIRST      = UINT64_C(1) << 21, /* we define one bit per resource limit here */
         SETTING_RLIMIT_LAST       = UINT64_C(1) << (21 + _RLIMIT_MAX - 1),
-        _SETTINGS_MASK_ALL        = (UINT64_C(1) << (21 + _RLIMIT_MAX)) - 1
+        _SETTINGS_MASK_ALL        = (UINT64_C(1) << (21 + _RLIMIT_MAX)) - 1,
+        _FORCE_ENUM_WIDTH         = UINT64_MAX
```

Comment:
btw, i figure one of those days we should probably rename this to _SETTINGS_MASK_FORCE_ENUM_WIDTH or so, i.e. add the namespacing prefix, as we might want to do the same for other enums too and then the names would clash